content/user/advanced-modules/PDFTemplates.adoc

```diff
@@ -0,0 +1,116 @@
+---
+Title: PDF-Templates
+Weight: 85
+---
+
+
+:toc:
+:toc-title: Table of content
+
+:experimental:
+
+:imagesdir: ./../../../../images/en/user
+
+ifdef::env-github[:imagesdir: ../../../../../master/static/images/en/user/PDFTemplates]
+
+:btn: btn:
+
+ifdef::env-github[:btn:]
+
+= PDF-Template
+
+This module allows you to create document templates. In the future, based on these templates, PDF documents or emails can be created, in which the necessary data from the SuiteCRM will automatically be added during the merge process.
+
+//image:image1.png[PDF-Template]
+
+
+== Creating the PDF-Template
+
+ .	In the module menu, select the option "Create PDF-Template".
+ .	In the Edit Form fill in the following fields:
+PDF-Template. :: Specify the name of the template being created.
+Creator. :: Please enter/ select the name of the creator of this template.
+Purpose. :: Here you should select the type of document to be created based on this template. By default, the system suggests creating templates for the following documents: Offers, Invoices, Contracts, Contacts or a Preliminary Contact.
+
+[cols="3,3",grid="none", frame="none"]
+|===
+a|{{% notice note %}}
+When adding a new item to the combo box list *_pdf_template_type_dom_*, the name of the key to be added must match the module name. In the following example, the item *Addresses* is added to the list. +
+Detailed information on adding a new element to the combo box is described in the
+Developer Tools page.
+{{% /notice %}}
+|image:Additiontocombobox.png[Adding a new item to the combo box]
+|===
+
+Sample From:: The system offers several options for ready-made templates, if necessary.
```

Comment:
Check the actual SuiteCRM UI for these strings, they need to match what is found in the app. This one is `Load sample`, for example.

Reply:
@jack7anderson7 I think you missed this comment, the strings are still the same...